NEWS.md

```diff
@@ -12,7 +12,13 @@
 
 ### Changed
 
-* Rename `binary` in `binary_outcome`
+* Add simulation experiments and corresponding visualizations.
+* Add `extract_effect_modifiers` function
+* Add `evaluate` function (for discovery evaluation)
+* Add `confounding` parameter (in `generate_cre_dataset`) 
+* Add `ite_pred` and `model` to CRE results
+* Add `binary_covariates` parameter (in `generate_cre_dataset`)
```

Comment:
Please update the news based on what is happening now. For example, in this case, it can be as follow:
`generate_cre_dataset` function now accepts `binary_covariate` boolean parameter as an input parameter.